pkg/upload/react-native-ios.go

```diff
@@ -132,7 +132,13 @@ func ProcessReactNativeIos(
 					plistPath = plistPathExpected
 					log.Info("Found Info.plist at: " + plistPath)
 				} else {
-					log.Info("No Info.plist found at: " + plistPathExpected)
+					plistPathExpected = filepath.Join(buildSettings.ProjectTempRoot, "ArchiveIntermediates/"+scheme+"/BuildProductsPath/Release-iphoneos/"+scheme+".app/Info.plist")
```

Comment:
```suggestion
					plistPathExpected = filepath.Join(buildSettings.ProjectTempRoot, "ArchiveIntermediates", scheme, "BuildProductsPath", filepath.Base(buildSettings.BuiltProductsDir), buildSettings.InfoPlistPath)				
```
* Better to just solely rely on comma-separation when using `filepath.Join` for compatibility. 
* Rather than explicitly set `Release-iphoneos` we should use what the value of `filepath.Base(buildSettings.BuiltProductsDir)` is, which I think should offer us a nice bit of flexibility.
* We can replace `scheme+".app/Info.plist"` with `buildSettings.InfoPlistPath` as well